admin/pages/accounts.php

```diff
@@ -495,7 +497,25 @@ class="fa fa-remove"></i> Cancel</span></a>
     };
     ?>
 </div>
-
+<?php 
+// List all accounts
```

Comment:
have you been tested with 2k accounts?

Reply:
I haven't tested it with this amount, but in the long term, this could cause crashes.

But since this is a temporary pull until version 1.0 of myacc comes out, I think it would be something that would help, but if necessary, I can review the code to load 100 accounts per page.

I actually did this because a community member asked me to, and I thought it would be useful to be able to bring it back, even though it's simpler.